### DISCUSSION TITLE
Validation of File Input

### DISCUSSION THREAD

**Discussion by jomellelastrilla** [jgile/vue-csv-import#76] [Q&A]
Is there a way to handle properly if the file input is empty and will not load/remove the mapping table.

It still load the previous csv fields even though I canceled the selection

![image](https://user-images.githubusercontent.com/2435077/130608238-6e0251cf-dafa-464a-99df-126d91ee4d63.png)

**Comment by James-Burgess** [comment]
As suggested [here](https://github.com/jgile/vue-csv-import/discussions/77#discussioncomment-1301420), you can re-render the VCI component after cancelling the selection via [key-changing](https://morioh.com/p/08963bf07353)
